app/Helpers/Helpers.php

```diff
@@ -131,6 +131,45 @@ function forceBibleisGideonsPagination($key, $limit_param)
     return [$limit, $is_bibleis_gideons];
 }
 
+function signedPath(
```

Comment:
how about we name this storagePath instead if signedPath. when used in conjunction with signedUrl, it is confusing